cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinAssociatedProject.java

```diff
@@ -0,0 +1,68 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import no.unit.nva.commons.json.JsonSerializable;
+import nva.commons.core.JacocoGenerated;
+
+import java.beans.ConstructorProperties;
+import java.net.URI;
+import java.util.Map;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CristinAssociatedProject implements JsonSerializable {
+
+    private final String cristinProjectId;
+    private final Map<String, String> title;
+    private final URI url;
+
+    @ConstructorProperties({"cristinProjectId", "title", "url"})
+    public CristinAssociatedProject(String cristinProjectId, Map<String, String> title, URI url) {
+        this.cristinProjectId = cristinProjectId;
+        this.title = title;
+        this.url = url;
+    }
+
+    public String getCristinProjectId() {
+        return cristinProjectId;
+    }
+
+    public Map<String, String> getTitle() {
+        return title;
```

Comment:
`nonEmptyOrDefault` ? search for it

Reply:
Maybe not relevant here since title will always have a value

Reply:
I better start using it!